aws/curator-ingress.yaml

```diff
@@ -1,40 +1,33 @@
 # Ingress defines how load balancers should forward traffic
 # from the internet traffic to the cluster.
 # https://kubernetes.io/docs/concepts/services-networking/ingress/
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
 kind: Ingress
 metadata:
-  name: curator-dev
+  name: curator
```

Comment:
nit: curator-ingress?

Reply:
I removed it on purpose because it felt redundant, I'm not adding -service or -deployment to the other objects so this is consistent.